### PR TITLE
Header Improvements

### PR DESCRIPTION
**Pull request by curtisupshall**
# Overview
Added header improvements from SIMS.

### CODE REVIEW

**Review by al-rosenthal — COMMENTED** [review]
I think @jeznorth updated the available headers with his last UI update. I think the header should just have Home, Submissions, Manage Users. Header changes look good otherwise

**Review by al-rosenthal — APPROVED** [review]
looks good 🎄

File: app/src/components/layout/header/Header.tsx [hunk 1]
```diff
@@ -1,27 +1,25 @@
-import { mdiHelpCircle } from '@mdi/js';
+import { mdiMenu } from '@mdi/js';
 import Icon from '@mdi/react';
-import { Theme } from '@mui/material';
+import { Menu, MenuItem, /*Theme*/ } from '@mui/material';
```
Comment (KjartanE) [inline app/src/components/layout/header/Header.tsx]: nit: remove

**Review by KjartanE — APPROVED** [review]
🐈 Ran locally, functioning properly

File: app/src/components/layout/header/Header.tsx (hunk 1 above)
Comment (curtisupshall) [reply app/src/components/layout/header/Header.tsx]: Good catch, removed

**Pull request merged**